attributes/default.rb

```diff
@@ -0,0 +1,73 @@
+#
+# Cookbook Name:: centreon-engine
+# Recipe:: default
+#
+# Copyright (c) 2016 The Authors, All Rights Reserved.
+
+#  --[ Centreon engine sudo configuration ]--
+default['centreon_engine']['sudo']['users'] = %w(centreon-engine centreon)
+
+# --[ Centreon engine home dir ]--
+default['centreon_engine']['user']['home_directory'] = "/var/lib/centreon-engine"
+
+# --[ Monitoring plugins dir ]--
+default['centreon_engine']['plugin_directory'] = "/usr/lib64/nagios/plugins"
+
+# --[ Centreon engine user parameters ]--
+default['centreon_engine']['user'] = "centreon-engine"
+default['centreon_engine']['group'] = "centreon-engine"
+
+# --[ Packages ]--
+default['centreon_engine']['server']['packages'] = %w(centreon-engine)
+
+# --[ Postfix ]--
+default['postfix']['main'] = {
```

Comment:
I don't think you want to mix up with postfix configuration here.
Either:
- don't touch postfix configuration in this cookbook (you'll do that in wrapper cookbook or simply using postfix cookbook default)
- extract this in a recipe that you conditionaly include in the default